### DISCUSSION TITLE
Welcome to RyzenTuner Discussions!

### DISCUSSION THREAD

**Discussion by zqhong** [zqhong/RyzenTuner#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by genji7430** [comment]
大佬用了这个之后游戏频率一直锁在了3ghz，之前正常都能跑到4ghz的，重装了bios删除了这个软件还是没恢复，自动模式就会一直频率反复跳，手动设置50w功率还是只能跑到26-28频率维持在3ghz

**Comment by zqhong** [comment]
重启一下。另外，记得在电源模式中开启睿频。
